setup.cfg

```
@@ -25,7 +25,7 @@ install_requires =
     matplotlib >= 2.2.0
     nibabel >= 3.0.1
     nipype >= 1.3.1
-    niworkflows ~= 1.1.6
+    niworkflows @ git+https://github.com/poldracklab/niworkflows.git@edeea81352f3f2b276f11e42e7829e1c5a03770f
```

```suggestion
    niworkflows ~= 1.1.7
```
though yet to be released

---

Let's do this with the release commit. No sense waiting on a PyPI release just to merge a PR.